test/integration/targets/s3_bucket_notification/tasks/main.yml

```diff
@@ -0,0 +1,329 @@
+---
+# ============================================================
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+# ============================================================
+- name: test add s3 bucket notification
+  block:
+    - name: move lambda into place for archive module
+      copy:
+        src: "function.zip"
+        dest: "{{output_dir}}/function.zip"
```

Comment:
We should not include a file of this size in the repo.  Use a very tiny lambda function and create the zip file during the test. If that's not possible, then we can put the zip file in our S3 bucket for CI -- but lets avoid that if possible.

Reply:
Ideally the lambda would be a singe python file with no requirements outside the standard library, then no zip would be needed. Keeping it python vs another language makes it easier to review since most contributors will be familiar with python.